frontend/js/components/map.js

```diff
@@ -139,15 +138,11 @@ export class LeafletMap extends React.Component {
             return <IssueMarker {...props} />
         })
 
-        let extents = geojsonExtent(JSON.parse(JSON.stringify(geojson)));
```

Comment:
This call is probably triggered by the `fetchIssueIfNeeded` action that is raised by `IssueDetailContainer` when the marker is clicked.

See https://github.com/mcallistersean/b2-issue-tracker/blob/master/frontend/js/actions.js#L106.